src/networks/migrations/0009_alter_network_network_type.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 5.1.5 on 2025-02-21 12:10
+
+from django.db import migrations, models
+
+OLD_NETWORK_TYPE = "social_and_community"
+NEW_NETWORK_TYPE = "social_and_sports"
+
+
+def run_network_type_update(apps, old_network_type, new_network_type):
+    Network = apps.get_model("networks", "Network")
+    Revision = apps.get_model("wagtailcore", "Revision")
+
+    for network_page in Network.objects.all():
+        if network_page.network_type == old_network_type:
+            network_page.network_type = new_network_type
+            network_page.save(update_fields=["network_type"])
+
+        revisions = Revision.objects.filter(
+            content_type_id=network_page.content_type.id, object_id=network_page.id
+        )
+        for revision in revisions:
+            if revision.content["network_type"] == old_network_type:
```

Comment:
Nico encountered an edge case where a network he has in his DB has a revision that was created before the `network_type` field.
To fix this he needed to add the following change so that we check the revision has that field in it's contents first, like so:
```suggestion
            if "network_type" in revision.content and revision.content["network_type"] == old_network_type:
```

Reply:
Done